src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -68,12 +68,15 @@
     public static final String INVALID_ADDRESS_DESC = " " + PREFIX_ADDRESS; // empty string not allowed for addresses
     public static final String INVALID_TAG_DESC = " " + PREFIX_TAG + "hubby*"; // '*' not allowed in tags
     public static final String INVALID_BIRTHDAY_DESC = " " + PREFIX_BIRTHDAY + "2020-Jan-12"; // only numbers
```

Comment:
Would `INVALID_BIRTHDAY_FORMAT_DISC` and `INVALID_BIRTHDAY_DATE_DESC` be better?